app/assets/javascripts/community_solution.js

```diff
@@ -21,8 +21,29 @@ $(document).on('turbolinks:load', function() {
         )
 
         $(document).on('theme:change:ace', CodeOceanEditor.handleAceThemeChangeEvent.bind(CodeOceanEditor));
-        $('#submit').one('click', CodeOceanEditorSubmissions.submitCode.bind(CodeOceanEditor));
-        $('#accept').one('click', CodeOceanEditorSubmissions.submitCode.bind(CodeOceanEditor));
+        $('#submit').one('click', submitCode.bind(CodeOceanEditor));
+        $('#accept').one('click', submitCode.bind(CodeOceanEditor));
     }
-
 });
+
+function submitCode(event) {
+    const button = $(event.target) || $('#submit');
+    this.startSentryTransaction(button);
+    this.teardownEventHandlers();
+    this.createSubmission(button, null, function (response) {
```

Comment:
Have you tested that these method calls are still successful? I would doubt that, since the `this.` reference changed.